docs/gaia/launch/blog-1-sp.md

```diff
@@ -0,0 +1,181 @@
+# Cosmos Hub para iniciar Mainnet
+
+## Dependencias previas al lanzamiento y cómo reclamar sus Atoms con seguridad
+
+En el verano de 2016, se publicó el [Cosmos whitepaper](https://cosmos.network/resources/whitepaper). En la 
+primavera de 2017, se completó la [recaudación de fondos de Cosmos](https://fundraiser.cosmos.network/). En los primeros 
+meses de 2019, el software está [completo](https://github.com/cosmos/cosmos-sdk/releases). El lanzamiento del Hub del Cosmos se acerca. 
+¿Qué significa esto para los poseedores de átomos?
+
+Si eres poseedor de Atoms, podrás delegar Atoms en los validadores de la red principal y votar sobre las propuestas de 
+gobernanza. De hecho, el éxito futuro de la red depende de que lo haga de forma responsable. Sin embargo, aún no podrás 
+transferir Atoms. Las transferencias se desactivarán a nivel de protocolo hasta que se ejecute un hard-fork para habilitarlas.
+
+Los poseedores de Atoms deben seguir cuidadosamente las directrices para poder delegar Atoms de forma segura. Por favor, 
+lea primero toda la guía para familiarizarse antes de hacer nada: [Guía CLI](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/gaia/delegator-guide-cli.md)
```

Comment:
Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.